app/assets/javascripts/controllers/middleware_datasource/middleware_datasource_service.js

```diff
@@ -32,6 +32,9 @@ function MwAddDatasourceService($http, $q) {
     {id: 'MYSQL', label: 'MySql', name: 'MySqlDS', jndiName: 'java:jboss/datasources/MySqlDS',
       driverName: 'mysql', driverModuleName: 'com.mysql', driverClass: 'com.mysql.jdbc.Driver',
       connectionUrl: '://localhost:3306/db_name'},
+    {id: 'OTHER', label: 'Other...', name: '', jndiName: '',
```

Comment:
unlike the other driver names that are more vendor specific, "Other" should probably be translation ready and not hardcoded